control/strategy/strategy.go

```diff
@@ -17,17 +17,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package routing
+package strategy
 
 import (
 	"time"
-)
 
-type SelectablePluginPool interface {
-}
+	"github.com/intelsdi-x/snap/core"
+)
 
 type SelectablePlugin interface {
 	HitCount() int
 	LastHit() time.Time
 	String() string
 }
+
+type RoutingAndCaching interface {
```

Comment:
Originally I had strategy.Strategy and felt it was better for readability to describe what kind of strategy was being provided.  

>I guess what I'm getting at is why not strategy.Strategy?

I lean towards being more descriptive.   Refactoring the name in the future can happen when and if there is a need for another strategy type.